README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+* Static variables
+* View Lifecycle (if available)
```

Comment:
Can we kill this? I hate this delineation, and it's only applicable for View Controllers and I think that it is a stupid to specify this just for them when we can have a better approach for _all_ class / struct orders:

I'd instead recommend this structure (including my suggestion above) based on my previous experience with C#:

`Public Properties`
` Static (public followed by private)`
`Private Properties`
`Init / Deinit`
`Override`
`Public Functions`
`Private Functions`